### PR TITLE
add constraint on generate submit button

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

**Comment by SOLenG** [comment]
See :
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1136

File: MediaAdminBundle/Form/Type/MediaCropType.php [hunk 1]
```diff
@@ -6,6 +6,9 @@
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
+use Symfony\Component\Form\FormView;
```
Comment (nicolasThal) [inline MediaAdminBundle/Form/Type/MediaCropType.php]: no blank lines in the uses

File: MediaAdminBundle/Form/Type/MediaCropType.php [hunk 2]
```diff
@@ -44,6 +47,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
+     * @param FormView      $view
+     * @param FormInterface $form
+     * @param array         $options
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options){
```
Comment (nicolasThal) [inline MediaAdminBundle/Form/Type/MediaCropType.php]: psr, the `{` should be on the next line

**Comment by nicolasThal** [comment]
Did you forgot to commit the test ?

**Pull request merged**